### PR TITLE
[156] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by OluwaboriShobayo**
image
image

image
image

image

### CODE REVIEW

File: submissions/oluwaboriShobayo/CSS/contactUs.css [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    text-align:center;
+    } 
+}
+
+
+ @media  only screen and (min-device-width: 600px) {
+     #header {
+            display:flex;
+            justify-content:space-between;
+            /* padding:20px;
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/CSS/contactUs.css:141]: I think you forgot to remove this comments

File: submissions/oluwaboriShobayo/CSS/productPage.css [hunk 2]
```diff
@@ -0,0 +1,237 @@
+/* This section is for the media screen query*/
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/CSS/productPage.css:1]: This task is for contact us page. Why are we have styling for product page? Kindly remove it

File: submissions/oluwaboriShobayo/CSS/signUp.css [hunk 3]
```diff
@@ -0,0 +1,289 @@
+
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/CSS/signUp.css:1]: This task is for contact us page. Why are we have styling for signup page? Kindly remove it

File: submissions/oluwaboriShobayo/HTML/Laptop media query.css [hunk 4]
```diff
@@ -0,0 +1,197 @@
+@media  only screen and (min-device-width: 1200px) {
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/HTML/Laptop media query.css:1]: What is this CSS style for? Meanwhile, all CSS file should be inside your CSS folder

File: submissions/oluwaboriShobayo/HTML/Laptopmedia-query.css [hunk 5]
```diff
@@ -0,0 +1,199 @@
+@media  only screen and (min-device-width: 1200px) {
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/HTML/Laptopmedia-query.css:1]: Same here too

File: submissions/oluwaboriShobayo/HTML/Mediaquer-Tab.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
+ @media  only screen and (max-device-width: 768px) {
```
Comment (adaugochi) [inline submissions/oluwaboriShobayo/HTML/Mediaquer-Tab.css:1]: And this too

**Review by adaugochi — CHANGES_REQUESTED** [review]
All images should be placed inside the `images` folder that is in your HTML folder

Also, remove spaces on file/folder names.

From what I am seeing it as if you did 3 tasks all at once. You are meant to do one task at a time

**Review by adaugochi — CHANGES_REQUESTED** [review]
These images are still in `html-forms` folder. Kindly move them to their appropriate folders 
image
Also. the images you use here is too much. are you sure you are using all of them?

Just so you know, you deleted your productPage.html. Is it intentional?
image

**Comment by OluwaboriShobayo** [comment]
Thanks Ada. I moved the productPage and the signUp page to another location because I felt that it might be causing confusion. I plan to add it back after the contactUs page has been merged.

Also, I have deleted the images I am not using.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**